mephisto/providers/mturk/utils/script_utils.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, TYPE_CHECKING
+
+from mephisto.providers.mturk.mturk_utils import give_worker_qualification
+from mephisto.data_model.requester import Requester
+
+if TYPE_CHECKING:
+    from mephisto.data_model.database import MephistoDB
+
+def direct_soft_block_mturk_workers(
+    db: "MephistoDB",
+    worker_list: List[str],
+    soft_block_qual_name: str,
+    requester_name: Optional[str] = None,
+):
+    """
+    Directly assign the soft blocking MTurk qualification that Mephisto 
+    associates with soft_block_qual_name to all of the workers in worker_list.
+    If requester_name is not provided, it will use the first mturk requester
```

Comment:
Hmm - is there a usecase for softblocking the first MTurk requester in the database? Otherwise it might be safer to check that `requester_name` is not None

Reply:
You can register as many of them as you want (such that if you're running different projects with different budgets or accounts, you can swap just by providing different names). In the case that you have multiple accounts, but don't specify  here, it'll use the most recently registered account to assign the qualification. Later on, assigning that qualification through mephisto (with `worker.grant_qualification`) will automatically use the correct account to grant from though.